packages/styles/src/link/index.css

```diff
@@ -6,13 +6,13 @@
  */
 .syn-link {
   /* The size of the icon */
-  --syn-link-icon-size: var(--syn-spacing-large);
+  --syn-link-icon-size: 1em;
 
   /* The gap that is used when there are pre or suffix icons */
   --syn-link-icon-gap: var(--syn-spacing-x-small);
```

Comment:
Can you please add a default gap according to `em` as well, pls? :)

Reply:
Sure, just set it to `0.25em`.

**Please note that the maximal amount of tokens for ChristianGTP is reached. Payment possible via Coffee** 😜